engine/test/integration_tests/dm_case_sensitive/run.sh

```diff
@@ -31,7 +31,7 @@ function run() {
 	echo "job_id: $job_id"
 
 	# wait for job finished
-	exec_with_retry --count 30 "curl \"http://127.0.0.1:10245/api/v1/jobs/$job_id\" | tee /dev/stderr | jq -e '.status == \"Finished\"'"
+	exec_with_retry --count 30 "curl \"http://127.0.0.1:10245/api/v1/jobs/$job_id\" | tee /dev/stderr | jq -e '.state == \"Finished\"'"
```

Comment:
Why did this test case pass before? I didn't find any changes related to changing `status` to `state`.

Reply:
`dm_case_sensitive` and `dm_full_mode` tests is not enabled before because they are waiting for some feature like `job detail`.

Reply:
They will enable these tests soon.